### ISSUE TITLE
Version number of re2c-2.0 is still 1.3

### ISSUE THREAD

**Issue by marryjianjian** [skvadrik/re2c#300]
https://github.com/skvadrik/re2c/blob/a058ba18fcfd8136a4c322de2260b35da07dc266/CMakeLists.txt#L2

I'm glad to see re2c-2.0 released (supported generate go! Wowwwww!!)
when I compile re2c-2.0 by myself, I wonder which verison I'm using. And I found `-v` prints 1.3
![image](https://user-images.githubusercontent.com/20366611/88820406-ddffac00-d1f3-11ea-8213-6a3e65bd499e.png)

Maybe someone just forgot. But as a new big version, it may cause misunderstanding.

**Comment by skvadrik** [comment]
Thanks! I fixed this in https://github.com/skvadrik/re2c/commit/9311e5295cac5ac76456ad78160fe0bd2f3ba2f0 (forgot to update the release script for the new CMake build system) and released minor bugfix release 2.0.1 to avoid confusion: https://re2c.org/releases/release_notes.html#release-2-0-1.

**Comment by marryjianjian** [comment]
Well, I don't know how release files packed. But it seems that [re2c-2.0.1.tar.xz](https://github.com/skvadrik/re2c/releases/download/2.0.1/re2c-2.0.1.tar.xz) 's files different from others ([2.0.1.zip](https://github.com/skvadrik/re2c/archive/2.0.1.zip) and [2.0.1.tar.gz](https://github.com/skvadrik/re2c/archive/2.0.1.tar.gz) ).
There is no CMake build system in [re2c-2.0.1.tar.xz](https://github.com/skvadrik/re2c/releases/download/2.0.1/re2c-2.0.1.tar.xz).

**Comment by skvadrik** [comment]
True, CMake files are missing from the release tarball re2c-2.0.1.tar.xz. This is a consequence of supporting two build systems: release tarball is packed with Autotools (`make dist`), and it does not need CMake files. I need to add them in `EXTRA_DIST`.

Zip files are different because they are packed automatically with Github (when I push release commit with a tag).

**Comment by marryjianjian** [comment]
Thanks! 
And I have another question about CMake build system in [Build instructions](https://re2c.org/install/install.html) which I didn't find it in this repository.
I found that `RE2C_BUILD_RE2GO` is needed rather than the option `RE2C_BUILD_GOLANG` mentioned in above instructions to build binary re2go. It seems it's typo but it does not affect use go backend with `--lang go`

**Comment by skvadrik** [comment]
Good catch, I corrected the error in documentation. I also pushed a fix that adds CMake files to the distribution tarball: https://github.com/skvadrik/re2c/commit/77da086ec3a35600135dc7309fd8cc1ab83c404a, and another fix that enables Go support by default: https://github.com/skvadrik/re2c/commit/ec654f12cb7934372d53f7045e95c2c683245cf7 (but it still can be turned off with a configuration option).

I plan to make a bugfix release 2.0.2 in about a week, to wait if any more bugs get reported. Thanks for your help!
